server/util/scratchspace/scratchspace.go

```diff
@@ -0,0 +1,54 @@
+package scratchspace
+
+import (
+	"flag"
+	"os"
+	"path/filepath"
+
+	"github.com/buildbuddy-io/buildbuddy/server/util/status"
+)
+
+var (
+	tempDir_doNotUseDirectly = flag.String("storage.tempdir", filepath.Join(os.TempDir(), "buildbuddy-scratch"), "Staging directory for ephemeral files. If unspecified, defaults to 'TEMPDIR/buildbuddy' where TEMPDIR is the OS-specific temp dir.")
```

Comment:
I don't think this needs to be named this way -- it's a package level private var you could just throw a comment on it?